spec/restaurant_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'restaurant'
+
+describe Restaurant do
+
+  it 'has a menu' do
+    expect(subject.view_menu).to eq "Burger: £9.99, Pizza: £6.99, Chips: £1.99, Salad: £8.99"
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)